src/components/footer/Footer.astro

```diff
@@ -0,0 +1,43 @@
+---
+import Socials from './Socials.svelte'
+---
+
+<footer>
+  <section class="bg-gray-750 py-6 text-white">
+    <div class="container mx-auto mt-6 px-4 sm:px-8">
```

Comment:
This could be simplified a bit :)

```suggestion
<footer class="bg-gray-750 py-6 text-white">
    <div class="container mx-auto mt-6 px-4 sm:px-8">
```

Reply:
Like
`<footer class="bg-gray-750 py-6 text-white">
    <div class="container mx-auto px-4 sm:px-8">`
? Or even more?